src/consensus/cost_calculator.py

```diff
@@ -47,10 +47,13 @@ def calculate_cost_of_program(
                 total_vbyte_cost += len(cvp_list) * ConditionCost.ASSERT_BLOCK_INDEX_EXCEEDS.value
             elif condition is ConditionOpcode.ASSERT_MY_COIN_ID:
                 total_vbyte_cost += len(cvp_list) * ConditionCost.ASSERT_MY_COIN_ID.value
-            elif condition is ConditionOpcode.ASSERT_COIN_CONSUMED:
-                total_vbyte_cost += len(cvp_list) * ConditionCost.ASSERT_COIN_CONSUMED.value
             elif condition is ConditionOpcode.ASSERT_FEE:
                 total_vbyte_cost += len(cvp_list) * ConditionCost.ASSERT_FEE.value
+            # TODO: give these real values
```

Comment:
This `TODO` should be moved to `condition_costs`.

Reply:
It can probably be deleted as most condition_costs are 0 at the moment

Reply:
fixed